src/components/member-profile/contribution-type/index.js

```diff
@@ -54,7 +65,22 @@ const ContributionType = (props) => {
       <div className={showMoreContentClass}>
         {type !== 'Active tasks' ? (
           <div>
-            {renderContributions(contributions, fullName, imageLink, devUser)}
```

Comment:
can we also take out this functions which are rendered directly with args something like
`const contributionInformation = {renderContributions(contributions, fullName, imageLink, devUser)}`
and then use the `contributionInformation` variable